src/main/java/com/cleanroommc/groovyscript/compat/mods/artisanworktables/Tables.java

```diff
@@ -0,0 +1,458 @@
+            ResourceLocation location = matchRequirementBuilder.getResourceLocation();
+            String modIdRequired = matchRequirementBuilder.getRequirementId().toLowerCase();
+
+            if (Loader.isModLoaded(modIdRequired)) {
+                this.requirements.put(location, matchRequirementBuilder.create());
+            } else {
+                issues.add(String.format("Mod %s required for %s was not loaded", modIdRequired, matchRequirementBuilder));
+            }
+
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription(field = "tools")
+        public ArtisanRecipeBuilder tool(IIngredient tool, int durability) {
+            tools.add(new ToolEntry(ArtisanIngredient.from(tool.toMcIngredient()), durability));
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder consumeSecondaryIngredients(boolean consume) {
+            consumeSecondaryIngredients = consume;
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder experience(int experience) {
+            this.experience = experience;
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder level(int level) {
+            this.level = level;
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder consumeExperience(boolean consume) {
+            consumeExperience = consume;
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder output(ItemStack stack) {
+            return output(stack, 1);
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder output(ItemStack stack, int weight) {
+            output.add(new OutputWeightPair(ArtisanItemStack.from(stack), weight));
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription(field = "extraOutputs")
+        public ArtisanRecipeBuilder extraOutput(ItemStack stack) {
+            return extraOutput(stack, 1.0f);
+        }
+
+        @RecipeBuilderMethodDescription(field = "extraOutputs")
+        public ArtisanRecipeBuilder extraOutput(ItemStack stack, float chance) {
+            extraOutputs.add(new ExtraOutputChancePair(ArtisanItemStack.from(stack), chance));
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder minimumTier(int minimumTier) {
+            this.minimumTier = minimumTier;
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder maximumTier(int maximumTier) {
+            this.maximumTier = maximumTier;
+            return this;
+        }
+
+        @RecipeBuilderMethodDescription
+        public ArtisanRecipeBuilder hidden() {
+            return hidden(true);

```

Comment:
should invert rather than set to be true

Reply:
Precedent for the current implementation btw:

![image](https://github.com/user-attachments/assets/92005bba-8e9f-453e-a652-3100f8192e36)